### PR TITLE
docs: fix typo in `specs/src/cat_pool.md`

### PR DESCRIPTION
**Pull request by sky-coderay**
## Overview

This PR fixes a typo in the `specs/src/cat_pool.md` file, changing `transacton` to `transaction`. This is important for maintaining accurate and professional documentation, ensuring clarity and preventing confusion for users and developers.

### CODE REVIEW

**Review by rootulp — APPROVED** [review]

**Comment by sky-coderay** [comment]
@rootulp Hi! Just a reminder about my PR . If everything looks good, it would be great if you could merge it. Thanks! 😊

**Comment by rootulp** [comment]
LGTM! We need one more approval before we can merge.

**Review by rach-id — APPROVED** [review]

**Pull request merged**
